frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx

```diff
@@ -18,6 +17,36 @@ export interface IAutocompleteBoxOption {
     label: string;
 }
 
+const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: theme.spacing(2),
+    '& .MuiInputLabel-root[data-shrink="false"]': {
+        top: 3,
+    },
+}));
+
+const StyledIcon = styled('div')<{ disabled: boolean }>(
```

Comment:
Proably should use **shouldForwardProp** as other similar components. Search from project for examples.